setup.cfg

```diff
@@ -100,3 +100,6 @@ ignore_missing_imports = True
 ; Supported with update
 [mypy-urllib3.*]
 ignore_missing_imports = True
+
+[mypy-moto.*]
```

Comment:
nit: can we keep this alphabetical? 

Also `Supported with update` means that a newer versions includes type hints, which I don't think is the case here?

Reply:
Done.